gpu4pyscf/df/df_jk.py

```diff
@@ -243,98 +241,101 @@ def to_cpu(self):
         obj = self.undo_df().to_cpu().density_fit()
         return utils.to_cpu(self, obj)
 
-def get_jk(dfobj, dms_tag, hermi=0, with_j=True, with_k=True, direct_scf_tol=1e-14, omega=None):
-    '''
-    get jk with density fitting
-    outputs and input are on the same device
-    TODO: separate into three cases: j only, k only, j and k
+def _jk_task_with_mo(dfobj, dms, mo_coeff, mo_occ, device_id,
```

Comment:
In this function and other helper functions, treat `device_id` as kwarg, and set its default value to 0